src/hacbs/components/ApplicationDetails/__tests__/HacbsApplicationDetails.spec.tsx

```diff
@@ -14,21 +15,51 @@ jest.mock('@openshift/dynamic-plugin-sdk-utils', () => ({
   useK8sWatchResource: jest.fn(),
 }));
 
+jest.mock('@redhat-cloud-services/frontend-components/useChrome', () => ({
+  useChrome: jest.fn(),
+}));
+
+const useChromeMock = useChrome as jest.Mock;
+
 configure({ testIdAttribute: 'data-test' });
 
 const watchResourceMock = useK8sWatchResource as jest.Mock;
 
 describe('HacbApplicationDetails', () => {
   it('should render spinner if application data is not loaded', () => {
     watchResourceMock.mockReturnValue([[], false]);
+    const set = jest.fn();
+    const toggle = jest.fn();
+    useChromeMock.mockReturnValue({
+      quickStarts: { set, toggle },
+    });
     render(<HacbApplicationDetails applicationName="test" />);
     expect(screen.queryByTestId('spinner')).toBeInTheDocument();
   });
 
   it('should render application display name if application data is loaded', () => {
     watchResourceMock.mockReturnValueOnce([mockApplication, true]);
+    const set = jest.fn();
+    const toggle = jest.fn();
+    useChromeMock.mockReturnValue({
+      quickStarts: { set, toggle },
+    });
     render(<HacbApplicationDetails applicationName="test" />);
     expect(screen.queryByTestId('details__title')).toBeInTheDocument();
     expect(screen.queryByTestId('details__title').innerHTML).toBe('Test Application');
   });
+
+  it('should only display getting started on first run', () => {
```

Comment:
Could you also add a test that asserts the quickstarts `toggle` api is called appropriately whether the localstorage value is set or not.

Reply:
yup, i added tests for set and toggle when localStorage is set or not 👍